.reuse/dep5

```diff
@@ -45,7 +45,7 @@ Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
 License: Apache-2.0
 
 Files: clients/fossid-webapp/src/test/assets/*
-Copyright: 2020-2021 Bosch.IO GmbH
+Copyright: 2020-2021 Bosch.IO GmbH <osm@bosch.com>
```

Comment:
Why is OSM@bosch.com added to the copyrights statement? Curious to know if this was decided somewhere.

Reply:
I had pinged @mkurzman quickly in a chat to ask for which general email address to use here, and that's simply what we cam up with. So it's not a fixed decision, but a proposal. Any objections?